onnxruntime/test/testdata/input_propagated_to_output.py

```diff
@@ -0,0 +1,87 @@
+"""
```

Comment:
Note: I tried to use the [onnx2py](https://github.com/microsoft/onnxconverter-common/blob/master/onnxconverter_common/onnx2py.py) tool to create this script from the model, but it threw some errors. So, I created this script manually.